docs/source/faq/faq.md

```diff
@@ -132,7 +132,7 @@ To later upgrade Kedro to a different version, simply run:
 pip install kedro -U
 ```
 
-When migrating an existing project to a newer Kedro version, make sure you also update the `project_version` in your `pyproject.toml` file from the project root directory or, for projects generated with Kedro<0.17.0, in your `ProjectContext`, which is found in `src/<package_name>/run.py`.
+When migrating an existing project to a newer Kedro version, make sure you also update the `kedro_init_version` in your `pyproject.toml` file from the project root directory or, for projects generated with Kedro<0.17.0, in your `ProjectContext`, which is found in `src/<package_name>/run.py`.
 
```

Comment:
This change is not really needed after #2373 is merged since this part is being removed.

Reply:
Thanks! The section is moving to the installation guide, so I have made the change as part of that PR 🙏